queries.sql

```diff
@@ -7,4 +7,42 @@ SELECT date_of_birth FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
 SELECT name, escape_attempts FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name != 'Gabumon';
-SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT species FROM animals;
+ROLLBACK;
+SELECT species FROM animals;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species LIKE 'unspecified';
+SELECT species from animals;
+COMMIT;
+SELECT species from animals;
+
+BEGIN;
+DELETE FROM animals
+SELECT COUNT(*) FROM ANIMALS;
+ROLLBACK;
+SELECT COUNT(*) FROM ANIMALS;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '01/01/2022';
+SAVEPOINT savepoint_1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO savepoint_1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(name) FROM animals;
+SELECT COUNT(name) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+
+SELECT SUM(escape_attempts) FROM animals WHERE neutered = true;
+SELECT SUM(escape_attempts) FROM animals WHERE neutered = false;
```

Comment:
-  [ ] Please, kindly instead of using 2 queries, try to answer the question with just 1 query.

## Hints

You could use MAX for this.

Example:

```SQL
SELECT neutered, MAX(escape_attempts) FROM animals
  GROUP BY neutered;
```

That way, with just 1 query we answer the question 💪 

_If you need help, I'm available on Slack_